cognite/neat/core/_data_model/importers/_rdf/_shared.py

```diff
@@ -99,31 +99,31 @@ def parse_properties(graph: Graph, query: str, language: str, issue_list: IssueL
         res["property_"] = quote(res["property_"], safe="")
         property_id = res["property_"]
 
-        # Safeguarding against incomplete semantic definitions
-        if not res["concept"] or isinstance(res["concept"], BNode):
+        # Skip Bnode
+        if isinstance(res["concept"], BNode):
```

Comment:
Do you have a guarantee that the user of this function passes in a query that returns a `concept`, `value_type`? Suggest doing a validation of that and gives some error messages.